L06-annotations/src/main/java/runner/TestRunner.java

```diff
@@ -0,0 +1,97 @@
+package runner;
+
+import annotations.After;
+import annotations.Before;
+import annotations.Test;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestRunner {
+
+
+    // Загружаем тестовый класс по имени
+    public static void runTests(String className) {
```

Comment:
К чему такие комментарии (здесь и для других методов)?
Так и называйте метод "Загружаем тестовый класс по имени" - `loadTestClassByName`.
Тогда никакие комментарии не нужны.

А если и пишете подобные комментарии, то лучше их оформлять как JavaDoc.
```suggestion
     /**
     * Загружаем тестовый класс по имени
     * @param className
     */
    public static void runTests(String className) {
```